src/lib/searchMessagesAPI.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import inatjs from "inaturalistjs";
+
+import MESSAGE_FIELDS from "../providers/fields";
+
+const searchMessages = async ( options: Object ): Promise<any> => {
+  const params = {
+    page: 1,
+    fields: MESSAGE_FIELDS
+  };
+
+  try {
+    const { results } = await inatjs.messages.search( params, options );
+    return results;
+  } catch ( e ) {
+    throw new Error( JSON.stringify( e.response ) );
+  }
+};
+
+export default searchMessages;
```

Comment:
I don't think each API method needs to be its own file, and I've never liked calling a directory `lib` unless it contains actual libraries. Can we call this directory `api` and maybe this file should be `messages.js`?